builtin/logical/pki/cert_util.go

```diff
@@ -124,12 +122,23 @@ func fetchCAInfo(req *logical.Request) (*caInfoBundle, error) {
 		return nil, errutil.UserError{Err: "backend must be configured with a CA certificate/key"}
 	}
 
-	var bundle certutil.CertBundle
-	if err := bundleEntry.DecodeJSON(&bundle); err != nil {
+	bundle := &certutil.CertBundle{}
+	if err := bundleEntry.DecodeJSON(bundle); err != nil {
 		return nil, errutil.InternalError{Err: fmt.Sprintf("unable to decode local CA certificate/key: %v", err)}
 	}
 
-	parsedBundle, err := bundle.ToParsedCertBundle()
+	return bundle, nil
+}
+
+// Fetches the CA info. Unlike other certificates, the CA info is stored
+// in the backend as a CertBundle, because we are storing its private key
+func fetchCAInfo(req *logical.Request) (*caInfoBundle, error) {
+	caBundle, err := fetchCABundle(req)
+	if err != nil {
+		return nil, errutil.InternalError{Err: fmt.Sprintf("unable to fetch CA bundle: %v", err)}
```

Comment:
I was waiting for more feedback since this was a simple one :), I knew there would be more.